GitHub_LearningTests/Feature/Login/Application/LoginControllerTests.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  LoginControllerTests.swift
+//  GitHub_LearningTests
+//
+//  Created by Amin Ashhab on 14.12.2020.
+//  Copyright © 2020 Amin_Second_Test_Project. All rights reserved.
+//
+
+import XCTest
+import CleanCore
+@testable import GitHub_Learning
+
+class LoginControllerTests: XCTestCase {
+    private var sut: LoginControllerImpl!
+    
+    private func setupTest(
+        dashboardLauncherControllerDummy: DashboardLauncherController = DashboardLauncherControllerDummy(),
```

Comment:
Tady nema byt `dashboardLauncherControllerDummy` ale jen `dashboardLauncherController`. Z pohledu toho controlleru to dummy neni.